fink_client/alertUtils.py

```diff
@@ -15,112 +15,83 @@
 # limitations under the License.
 import os
 import io
-import confluent_kafka
 import json
 import glob
-import time
 import fastavro
 from typing import Iterable
 
 def read_avro_alerts(data_path: str) -> Iterable[dict]:
-    """ Read avro alert files and return an interable 
+    """ Read avro alert files and return an interable
     with dicts of alert data
-    
+
     Parameters
     ----------
     data_path: str
         a directory path where to look for avro alert files
-        
+
     Returns
     ----------
     record: Iterable
         a generator that yields records(dict) after reading avro files
```

Comment:
Typo: there should be a space in between records and dict?

Reply:
fixed in 7be307a.